api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -104,7 +119,14 @@ const (
 // DRPlacementControlSpec defines the desired state of DRPlacementControl
 type DRPlacementControlSpec struct {
 	// PlacementRef is the reference to the PlacementRule used by DRPC
-	PlacementRef v1.ObjectReference `json:"placementRef"`
+	PlacementRef v1.ObjectReference `json:"placementRef,omitempty"`
+
+	// PrimaryProtectedNamespace is the namespace where the application is running on the dr cluster.
+	// This field is mandatory when the DRPC is created for discovered or imperative apps.
+	PrimaryProtectedNamespace string `json:"primaryProtectedNamespace,omitempty"`
```

Comment:
Do we need 2 namespace fields, can we just use `EligibleNamespaces` as a single field and use it for required purposes?

The Recipe reference would be from one of the eligible namespaces